### PR TITLE
feat(helm): allow passing in tolerations per deployment

### PR DESCRIPTION
**Pull request by tcrasset**
Tolerations (on pods) allows pods to be scheduled on nodes with a taint.
If a specific pod doesn't have the corresponding tolerations to a tainted node, it won't be scheduled.

This is used so that we can better control which pods go where in our cluster, without relying on an implicit coupling of the `highMemory` key being present.

In our pulumi program, we can now pass in the `tolerations` manually if we want.

### CODE REVIEW

**Review by tas17 — APPROVED** [review]
LGTM!

**Review by chybz — APPROVED** [review]

**Review by lusehair — APPROVED** [review]
Seems ok for me :)

File: helm-chart/values.yaml [hunk 1]
```diff
@@ -64,22 +67,36 @@ debug:
...
+  # If true, the worker will use KEDA to autoscale the number of pods based on the number of tasks in the queue.
+  useKedaAutoscaler: false
+
+  # The following values are used to set the configuration options for the autoscaling.
+  # Only used if useKedaAutoscaler is true.
   scaling:
     maxReplicaCount: 10
     scaledComponents:
       - identifier: "medium"
         preferredNodeType: ""
```
Comment (ykatchou) [inline helm-chart/values.yaml:84]: No small ?
I recall we use two select between the 3, not only between two.
what will happen for small calculations ? they remains on keda triggering ?

Comment (tcrasset) [reply helm-chart/values.yaml:84]: The small is not autoscaled using keda.

**Pull request merged**